compiler-lib/compiler/commands/test.rkt

```diff
@@ -666,7 +666,13 @@
   ;; This should be in Racket somewhere and return all the collection
   ;; paths, rather than just the first as collection-path does.
   (define (collection-paths c)
-    (match-define (list-rest sc more) (map path->string (explode-path c)))
+    (define (path->string* maybe-path)
```

Comment:
Okay I'll go make a PR there and see what people think. If that gets accepted then we can just ignore all of this.